twitter_analysis/app.py

```diff
@@ -151,7 +152,7 @@ def convertData():
     for i in topics:
```

Comment:
use more descriptive variables. Eg, this should read, "for topic in topics_list" or something like this. Make it as human readable as possible